sdk/azcore/CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Release History
 
+## 1.8.0 (2023-10-05)
+
+### Features Added
+
+* Added `Claims` and `TenantID` fields to `policy.TokenRequestOptions`.
```

Comment:
`TenantID` was in a prior stable version
```suggestion
* Added `Claims` and `EnableCAE` fields to `policy.TokenRequestOptions`.
```

Reply:
I can't copy/paste :p.  Fixed (also removed the duplicate entry).